client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -15,22 +14,19 @@ import FollowButton from 'reader/follow-button';
 import { getStreamUrl } from 'reader/route';
 import EmailSettings from './email-settings';
 
-const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( '/', '' );
-
 function ReaderSubscriptionListItem( {
 	url,
 	feedId,
 	feed,
 	siteId,
 	site,
 	className = '',
-	followSource,
-	lastUpdated,
 	translate,
+	followSource,
 } ) {
-	const siteTitle = feed && feed.name;
+	const siteTitle = ( feed && feed.name ) || ( site && site.title );
```

Comment:
maybe we could use `siteNameFromSiteAndPost` from `reader/utils` but enhance it to take a feed too?